frontend/widgets/OBSBasic.hpp

```diff
@@ -1456,6 +1456,13 @@ private slots:
 	std::vector<OBSDataAutoRelease> safeModeTransitions;
 	QPointer<QPushButton> transitionButton;
 	QPointer<QMenu> perSceneTransitionMenu;
+
+	/* NOTE: There is a reliance on the order of insertion */
+	std::map<std::string, OBSSource> transitions;
+	/* FIXME: Replace usages of a name to identify a transition */
+	std::unordered_map<std::string, std::string> trsNameUuid;
```

Comment:
```suggestion
	std::unordered_map<std::string, std::string> transitionUuidToNames;
```
Let's use expressive names that carry semantic as well as functional meaning.

Reply:
Renamed, it was meant to be removed but it requires a bigger and heavier work to get rid of it.